templates/tcpwrappers.allow.erb

```diff
@@ -5,8 +5,8 @@ lpattern = Array.new
 Array(@pattern).flatten.each do |ptn|
   if ptn == 'any'
     lpattern.push('ALL')
-  elsif ptn !~ /[[:alpha:]]/
-    lpattern.push(scope.function_bracketize([scope.call_function('simplib::nets2ddq', [ptn])]))
+  elsif ptn !~ /^[[:alpha:]]*$/
```

Comment:
IPv6 addresses can start with letters, so this won't pass all IPv6 addresses into the underlying functions for bracketization

Reply:
I think both simplib::nets2ddq and simplib::bracketize return non IP strings untouched.  So you may not to have any check at all and just pass it to the nested function calls.

Reply:
Yes it will @trevor-vaughan , it is going into that section when it is not a word made of only A-Za-z. Therefore any ip (with '.' or ':') is going to go in there, as well as lists of hosts and/or ips. That is why the previous was failing, it was just considering if there existed alpha there .

And yes, @lnemsick-simp you're correct.  I was figuring the previous intention of the split like this was to streamline it if the user only had a single hostname, which is why I was keeping it there. I'm totally cool taking out the extra else.